src/makeRobotAccountant.js

```diff
@@ -1,12 +1,29 @@
-'use strict';
+"use strict";
 
 /**
  *
  * @return {function}
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCount = 0;
+  const errorMessage = "Bzzz... Error!";
+
+  function getSum(a) {
+    return function (b) {
+      callCount++;
+
+      if (callCount <= 3) {
```

Comment:
if you'd change this codition you could have saved 1 if statement 😄 
```suggestion
      if (callCount > 3 && callCount % 2 === 0) {
        return a + b
```